csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -3352,6 +3350,125 @@ This does not exclude that such a document is also available in an access protec
 
 > Reasoning: If an advisory is already in the media, an end user should not be forced to collect the pieces of information from a press release but be able to retrieve the CSAF document.
 
+### 5.1.5 Requirement 5: security.txt
+
+In the security.txt there MUST be at least one field `CSAF` which points to either the ROLIE service document or a directory with CSAF files. If this field indicates a web URI, then it MUST begin with "https://" (as per section 2.7.2 of [RFC7230]).
```

Comment:
Perhaps this could refer to https://securitytxt.org/ for more information about security.txt files.

Reply:
@sthagen: Please adopt the format if needed.